astracliGettingStarted/README.md

```diff
@@ -7,7 +7,7 @@ The DataStax Astra Command Line Interface (CLI) is a tool for both Astra DB and
 - Use the CLI to connect to and manage Astra DB
 
 # Prerequisites
-To use the Astra CLI, you need to create a token with the _"Organization Administrator"_ role.
+To use the Astra CLI, you need to create a token with the **Organization Administrator** role.
```

Comment:
**Ignore this update--I saw how you edited the other roles. This matches just fine.